src/main/java/org/mskcc/cmo/metadb/service/impl/ValidRequestCheckerImpl.java

```diff
@@ -243,6 +316,7 @@ private boolean hasSampleType(Map<String, String> sampleMap)
         if (Strings.isBlank(sampleType)) {
             return hasNAtoExtract(sampleMap);
         }
+        //are baitset and recipe same here?
```

Comment:
these are not assumed to be the same - regardless though it's `baitSet` which is used in the sample label generation

Reply:
remove comment here